### PR TITLE
FEATURE: BEND-29-delete-language-by-id

### PR DESCRIPTION
**Pull request by Kehomaxd**

### CODE REVIEW

File: src/http/common/Base/static/Base.UserEndpoints.ts [hunk 1]
```diff
@@ -18,4 +18,5 @@ export enum UserEndpoints {
   GetEducationById = '/user/:userId?/education/:educationId',
   GetWorkById = '/user/:userId?/works/:workId',
   Delete = '/user/:userId?',
+  DeleteLanguage = '/user/:userId?/language/:languageId',
```
Comment (octapf) [inline src/http/common/Base/static/Base.UserEndpoints.ts:21]: @whitehatdevv should the params always be optional? Ex: 
 :languageId?

File: src/models/model.language.ts [hunk 2]
```diff
@@ -8,6 +8,11 @@ const Schema = mongoose.Schema;
  */
 
 const LanguageSchema = new Schema({
+  _id: {
```
Comment (octapf) [inline src/models/model.language.ts:11]: @whitehatdevv does not mongoose create the _id field automaticly?

File: src/http/actions/User.ts [hunk 3]
```diff
@@ -490,11 +490,46 @@ export module UserActions {
       .catch(_ =>
         f([
           {
-            message: ErrorMessages.AppUserServerWasUnableDeleteUser,
+            message:
+              ErrorMessages.AppUserUseridLanguagesServerWasUnableProcessData,
```
Comment (octapf) [inline src/http/actions/User.ts:494]: @Kehomaxd the UUID name must match the ErrorsMessage name

Comment (OvidioSantoro) [reply src/http/actions/User.ts:494]: I think this error message is from the previous function `DeleteAccountAction` and was changed by mistake.

Comment (Kehomaxd) [reply src/http/actions/User.ts:494]: yeah i gonna fix this

**Comment by whitehatdevv** [comment]
@Kehomaxd @PatyVilas please fix the conflicts

**Pull request merged**